### ISSUE TITLE
如何在YOLOX程序中区分出验证集和测试集

### ISSUE THREAD

**Issue by xmayyy** [Megvii-BaseDetection/YOLOX#1317]
如何在YOLOX程序中区分出验证集和测试集

**Comment by FateScript** [comment]
Validation set and test set is defined by your task. I'm not sure what you are asking.

**Comment by xmayyy** [comment]
在YOLOx的论文中，有val的map，也有test的map，想知道是在代码的哪里和输出终端的哪里区分出来val的map和test的map的

**Comment by FateScript** [comment]
COCOEval only print mAP of evalution. As for mAP of testing, we submitted json file to official website of coco  and  get a mAP value.

**Comment by xmayyy** [comment]
那VOC数据集如何操作，在VOC数据集中测test和val的map,训练完是会有个
![image](https://user-images.githubusercontent.com/79397109/168950616-ef2ab06d-b7cf-418c-a6dd-b782af39661b.png)
这样的文件，我应该使用它怎么去测map吗

**Comment by Astroooh** [comment]
> 

您好，在训练过程中，打印的是验证集的mAP，那我如果想用训练后的权重去评估测试集mAP，需要怎么去做呢，我用的是自定义的VOC格式的数据集，刚刚接触请多多指教～

**Comment by Astroooh** [comment]
> COCOEval only print mAP of evalution. As for mAP of testing, we submitted json file to official website of coco and get a mAP value.

您好，在训练过程中，打印的是验证集的mAP，那我如果想用训练后的权重去评估测试集mAP，需要怎么去做呢，我用的是自定义的VOC格式的数据集，刚刚接触请多多指教～

**Comment by MengtianLee** [comment]
> > COCOEval only print mAP of evalution. As for mAP of testing, we submitted json file to official website of coco and get a mAP value.
> 
> 您好，在训练过程中，打印的是验证集的mAP，那我如果想用训练后的权重去评估测试集mAP，需要怎么去做呢，我用的是自定义的VOC格式的数据集，刚刚接触请多多指教～

You could try: python eval.py --c tha path of trained model -f yolox_s.py --conf 0.01 --nms 0.65 (all parameters should be equal to the parameters setting during training.)
If you want to evaluate on the test data, you should add "--test". (default: val)

**Comment by bling-bling-only** [comment]
但是加上--test，结果输出跟不加一样
